to.etc.domui/src/main/java/to/etc/domui/util/exporters/ExcelExportWriter.java

```diff
@@ -367,18 +385,18 @@ public void renderCell(ExcelExportWriter<?> exporter, Cell cell, int cellIndex,
 		register(byte.class, NUMBER_CONVERTER);
 		register(short.class, NUMBER_CONVERTER);
 		register(int.class, NUMBER_CONVERTER);
-		register(long.class, NUMBER_CONVERTER);
-		register(float.class, NUMBER_CONVERTER);
+		register(long.class, DECIMAL_CONVERTER);
```

Comment:
long cannot have a fraction, it should remain NUMBER_CONVERTER.

Reply:
ofc